stl/inc/algorithm

```diff
@@ -207,6 +212,27 @@ __declspec(noalias) void _Replace_vectorized(
     }
 }
 
+template <class _Ty>
+_Ty* _Unique_vectorized(_Ty* const _First, _Ty* const _Last) noexcept {
+    if constexpr (is_pointer_v<_Ty>) {
+#ifdef _WIN64
+        return reinterpret_cast<_Ty*>(::__std_unique_8(_First, _Last));
+#else
+        return reinterpret_cast<_Ty*>(::__std_unique_4(_First, _Last));
+#endif

```

Comment:
Unlike other vectorized algorithms, it looks like these cases aren't necessary. I'd recommend dropping them, and ensuring that we have test coverage of unique-ifying pointer elements (it doesn't need to be exhaustive, just enough to verify that it works, then the normal test coverage for 4 and 8-byte integers will suffice).